.github/workflows/cron-job-templating/cron-template.j2

```diff
@@ -0,0 +1,177 @@
+name: cron-{{ input_epoch }}-core-{{ input_os }}-latest
+
+env:
+  os: {{ input_os }}-latest
+  distro: core
+  epoch: '{{ input_epoch }}'
+
+
+{{ 'on:' }}
+  schedule:
+    # every other day
+    - cron: '37 3 */2 * *'
+
+jobs:
+  install_env:
+    name: "install qiime2-core={{ input_epoch }} ({{ input_os }}-latest)"
+    runs-on: {{ input_os }}-latest
+    {% raw %}
```

Comment:
Please drop all of the `{% raw %}` tags (including the closing tags) once you have overridden the template syntax.